include/open62541cpp/open62541client.h

```diff
@@ -540,6 +607,27 @@ class Client
         return lastOK();
     }
 
+    /*!
+        \brief connect
+        \param endpointUrl
+        \return true on success
+    */
+    bool connectNoInit(const std::string& endpointUrl)
+    {
+        // initialise();
```

Comment:
could also be implemented as an argument (default to "init = true") instead of an extra method? extra method seems unnecessarily verbose

Reply:
agree, changed this on: https://github.com/psiori/open62541Cpp/pull/5